src/main/scala/temple/builder/ServerBuilder.scala

```diff
@@ -30,18 +30,25 @@ object ServerBuilder {
       case ServiceLanguage.Go => GoLanguageConfig
     }
 
-    val createdBy: CreatedByAttribute = serviceBlock.lookupMetadata[ServiceEnumerable] match {
+    // TODO: This could be nicer
+    val createdBy: Option[CreatedByAttribute] = serviceBlock.lookupMetadata[ServiceEnumerable] match {
```

Comment:
There's actually quite a lot you can do here. If you replace the `match` with a `map`, you can remove the `None => None` case as well as the `Some` wrapping on the `CreatedByAttribute`. Then because both ServiceEnumerable cases are the same except for a boolean, we can just make the pattern match using a variable and use this to create the attribute.

All in all I think this looks like:

```scala
  val createdBy: Option[CreatedByAttribute] = serviceBlock.lookupMetadata[ServiceEnumerable].map {
      case ServiceEnumerable(filterByThis) =>
        CreatedByAttribute(
          languageConfig.createdByInputName,
          languageConfig.createdByName,
          filterEnumeration = filterByThis,
        )
    }
```